### DISCUSSION TITLE
Use Chrome OS Flex image with Brunch

### DISCUSSION THREAD

**Discussion by Rrishik** [sebanc/brunch#1504] [General]
Hi, I tried using the Chrome Os Flex image with Brunch for dual booting on my Dell (Legacy BIOS) laptop following the [Linux Dual Booting](https://github.com/sebanc/brunch/blob/master/install-with-linux.md#dualboot-installations) guide alongside Manjaro.

I just wanted to let everyone know that it works!. This will be helpful for people who are willing to sacrifice the Android app support and just want to try out the Chrome OS Flex alongside their existing installations.

Thanks!

**Comment by darethehair** [comment]
Cool!  Does the 'menuentry' for Flex look very similar to the one used for Brunch?  Could you show it here?

**Reply by Rrishik** [reply]
Yeah it looks the same.

```
menuentry "ChromeOS" --class "brunch" {
	img_path=/chromeos.img
	img_uuid=60550890-03
	search --no-floppy --set=root --file $img_path
	loopback loop $img_path
	source (loop,12)/efi/boot/settings.cfg
	if [ -z $verbose ] -o [ $verbose -eq 0 ]; then
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug img_uuid=$img_uuid img_path=$img_path \
			console= vt.global_cursor_default=0 brunch_bootsplash=$brunch_bootsplash quiet
	else
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug img_uuid=$img_uuid img_path=$img_path
	fi
	initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
}
```
![PXL_20220412_161738208 NIGHT](https://user-images.githubusercontent.com/12758277/163108329-1c9403c6-b317-4610-98df-b7321bc6ae15.jpg)
![PXL_20220413_052034390](https://user-images.githubusercontent.com/12758277/163108506-bbb55398-2ae3-4f61-9d55-d7bcac0636a1.jpg)

**Comment by darethehair** [comment]
Thanks for the info and pics!
